### PR TITLE
Align Versions to Right

### PR DESCRIPTION
**Pull request by daming-lu**
https://github.com/PaddlePaddle/PaddlePaddle.org/issues/432

The result looks like this:

screen shot 2018-03-05 at 5 33 37 pm

### CODE REVIEW

File: portal/portal/static/sass/navigation.sass [hunk 1]
```diff
@@ -371,4 +371,7 @@ body
 
 .modal-button-group
     a, a:visited
-        color: black
\ No newline at end of file
+        color: black
+
+.version-align-right
+    padding-left: 66%
```
Comment (jetfuel) [inline portal/portal/static/sass/navigation.sass]: Can we try `  text-align: right;` ? or just `align: right` ? `padding-left` should be used to give a small margin to the left, but for aligning the whole block to the right.

Comment (daming-lu) [reply portal/portal/static/sass/navigation.sass]: screen shot 2018-03-06 at 2 12 10 pm

`text-align: right` will not align the words.  Is `align: right` valid?

**Comment by cs2be** [comment]
why are we aligning to right?

**Comment by jetfuel** [comment]
@cs2be There is a ticket for it. https://github.com/PaddlePaddle/PaddlePaddle.org/issues/432

**Pull request merged**